wled00/FX_fcn.cpp

```diff
@@ -1398,7 +1398,7 @@ void WS2812FX::service() {
     _lastServiceShow = nowUp; // correct timestamp, for better FPS control
   }
   #ifdef WLED_DEBUG
-  if (millis() - nowUp > _frametime*2) DEBUG_PRINTF_P(PSTR("Slow strip %u/%d.\n"), (unsigned)(millis()-nowUp), (int)_frametime);
+  if ((_targetFps != FPS_UNLIMITED) && (millis() - nowUp > _frametime)) DEBUG_PRINTF_P(PSTR("Slow strip %u/%d.\n"), (unsigned)(millis()-nowUp), (int)_frametime);
```

Comment:
This will end up in non-debug builds like `if (...) ;` which is a waste of CPU cycles.
Wrap `#ifdef` around it.

Reply:
aehm ... not sure what you refer to ... its already wrapped
```C++
#ifdef WLED_DEBUG
    if ((_targetFps != FPS_UNLIMITED) && (millis() - nowUp > _frametime)) DEBUG_PRINTF_P(PSTR("Slow strip %u/%d.\n"), (unsigned)(millis()-nowUp), (int)_frametime);
#endif
```

Reply:
Uh. I'm sorry. I've become sloppy.